### ISSUE TITLE
Menu: Section headers should be read out by screen readers

### ISSUE THREAD

**Issue by eddysims** [GetJobber/atlantis#698]
Section headers are not read out to assistive technologies.

**Comment by Bhavin789** [comment]
Hi @eddysims , I was going through this issue. Are we talking about the section headers in `Menu` component? Any example of other components? Thanks

![image](https://user-images.githubusercontent.com/15359575/135725466-267aeb7b-123f-47d4-99c5-9f3d2860127b.png)
